extensions/amp-story/1.0/test/test-amp-story-progress-bar.js

```diff
@@ -30,20 +31,26 @@ describes.realWin('amp-story-progress-bar', {amp: true}, (env) => {
       'page-4',
       'page-5',
     ]);
-    const progressBar = ProgressBar.create(env.win, storyEl).build('page-1');
-    doc.body.appendChild(progressBar);
+    progressBar = ProgressBar.create(env.win, storyEl);
+    doc.body.appendChild(progressBar.build('page-1'));
   });
 
   describe('story ad progress segment', async () => {
-    it('should create/remove ad segment based on ad visibility', () => {
+    it('should create progress bar with ad pages', () => {
```

Comment:
Please add another test that checks that there's the correct count of segments in the progress bar

Reply:
Done